src/System.Linq/src/System/Linq/Iterator.cs

```diff
@@ -13,8 +12,8 @@ public static partial class Enumerable
         internal abstract class Iterator<TSource> : IEnumerable<TSource>, IEnumerator<TSource>
         {
             private int _threadId;
-            internal int state;
-            internal TSource current;
+            internal int _state;
+            internal TSource _current;
```

Comment:
This name change brings me more happiness than a rational person should find in such a thing :smile: